src/core/meshcop/tcat_agent.cpp

```diff
@@ -102,12 +98,22 @@ void TcatAgent::Stop(void)
 {
     mCurrentApplicationProtocol = kApplicationProtocolNone;
     mState                      = kStateDisabled;
-    mAlreadyCommissioned        = false;
     mAppDataReceiveCallback.Clear();
     mJoinCallback.Clear();
     LogInfo("TCAT agent stopped");
 }
 
+Error TcatAgent::SetTcatVendorInfo(const VendorInfo &aVendorInfo)
+{
+    Error error = kErrorNone;
+    VerifyOrExit(aVendorInfo.IsValid(), error = kErrorInvalidArgs);
```

Comment:
Newline after variable declarations?
```suggestion
    Error error = kErrorNone;
    
    VerifyOrExit(aVendorInfo.IsValid(), error = kErrorInvalidArgs);
```